redirect/index.php

```diff
@@ -2,5 +2,16 @@
 
 include __DIR__ . '/routes.php';
 $routeHandler = new Route();
-header('Location: ' . $routeHandler->getTargetOfSub($_SERVER['HTTP_HOST'])); // Function does check if GET parameter is set / empty
+$target = $routeHandler->getTargetOfSub($_SERVER['HTTP_HOST']);// Function does check if GET parameter is set / empty
+
+if($routeHandler->shouldAddReferer($_SERVER['HTTP_HOST'])) {
+    if(str_contains($target, '?')) {
+        $target .= '&'; // add & if there is a query variable in the target
+    }else {
+        $target .= '?'; // ? otherwise
+    }
+    $target .= 'r=' . urlencode($_SERVER['REQUEST_URI']);
```

Comment:
Can we please just add this for trusted domains? (As a list) I don't want a random GET parameter attached to all URLs. It might cause problems.

Reply:
Ah nevermind, you already did that :dancers: